### DISCUSSION TITLE
SqlTask导致数据库连接数被打满

### DISCUSSION THREAD

**Discussion by nizhouwen** [apache/dolphinscheduler#13996] [Q&A]
SqlTask类型的任务，执行的时候创建了一个数据源连接池HikariDataSource，怎么没看到关闭这个连接池，Doris连接数被打满了

**Comment by xuhui1231** [comment]
我也遇到这个问题了我的是hive被打满了，请问你你解决了吗

**Comment by zhongjiajie** [comment]
already have a issue https://github.com/apache/dolphinscheduler/issues/14179 and will track in that issue instead of discussion
